src/features/courses/course-controller.ts

```diff
@@ -439,14 +439,15 @@ class CourseController {
             courseId,
             questionId,
             topicId,
-            unitId
+            unitId,
+            userId,
         } = options.where;
 
         const setFilterCount = [
             courseId,
             questionId,
             topicId,
-            unitId
+            unitId,
```

Comment:
You didn't add userId here? should it be?  maybe a comment as to why not

Reply:
If this is the case it is saying that you can't get all grades for a user for all courses which might be alright